### PR TITLE
[refs #293] Preserve centered items when using flex

### PR DESCRIPTION
**Pull request by zomars**

### CODE REVIEW

**Comment by herzinger** [comment]
sounds good

File: objects/_objects.layout.scss [hunk 1]
```diff
@@ -211,6 +211,10 @@ $inuit-use-markup-fix: false !default;
     display: flex;
   }
 
+  &.o-layout--center {
+    justify-content: center;
+  }
```
Comment (anselmh) [inline objects/_objects.layout.scss]: This notation is a bit awkward, to be honest. It’d only be available when `o-layout--stretch` is set on the same element. This to me is a bit to unflexible. By the way I find it a bit weird to have `o-layout--stretch o-layout--center` on the same element. I’d assume both are for different things, not an add-up class.

Comment (anselmh) [inline objects/_objects.layout.scss]: That said, I find `o-layout--stretch` not very convincing here now as stretch suggests `align-items/content: stretch`. @inuitcss/core any thoughts?

Comment (zomars) [inline objects/_objects.layout.scss]: My intention here was to preserve the original behavior for when using `.o-layout--center` by itself. The stretch modifier stretches in a vertical way, so a center alignment can be necessary at times.

**Comment by csshugs** [comment]
@inuitcss/core 
I'm just having a look at this PR again. Isn't this PR obsolete, because #308 is already merged?

**Comment by csshugs** [comment]
> I'm just having a look at this PR again. Isn't this PR obsolete, because #308 is already merged?

I just figured out myself:
Yes!

**Pull request closed without merging**